src/js/conduit/Router.test.js

```diff
@@ -2,6 +2,124 @@ import '@babel/polyfill';
 
 import Router from './Router';
 
-test('adds 1 + 2 to equal 3', async () => {
-  expect(1 + 2).toBe(3);
+describe('Router', () => {
+  describe('export value', () => {
+    test('is function', () => {
+      expect(typeof Router).toBe('function');
+    });
+  });
+
+  describe('has everything necessary for iframe functionality', () => {
+    test('returns object', () => {
+      const router = Router();
+
+      expect(typeof router).toBe('object');
+    });
+    test('has "Router" property, which is the Router class', () => {
```

Comment:
this is pretty descriptive maybe even change `"Router"` to `.Router` to be close to code.

Reply:
I ended up consolidating the individual property assertions into a single test